sihl/test/web.ml

```diff
@@ -1,6 +1,62 @@
 open Lwt.Syntax
```

Comment:
Please add tests here for empty route strings, empty lists in `combine` nestings, and other edge cases

Reply:
Done

Reply:
Thanks